packages/cli/src/models/local/UnpackController.ts

```diff
@@ -0,0 +1,72 @@
+import path from 'path';
+import request from 'request-promise-native';
+import fs from 'fs-extra';
+
+import { Logger } from 'zos-lib';
+
+import stdout from '../../utils/stdout';
+import patch from '../../utils/patch';
+import cp from '../../utils/cp';
+
+const ora = patch('ora');
+const simpleGit = patch('simple-git/promise');
+
+interface KitConfig {
+  ignore: string[],
+  hooks: Object,
+}
+
+export default class UnpackController {
+  public async unpack(url: string, workingDirPath: string = '', config: KitConfig): Promise<void | never> {
+    if (!url) throw Error('A url must be provided.');
+    if (!config) throw Error('A config must be provided.');
+
+    const { exec } = cp;
+    const { readdir, remove, pathExists } = fs;
```

Comment:
If we remove the need for `remove` (pun not intended), could we also remove (still not intended) `fs-extra` and just use regular `fs`, so we don't introduce another dependency?

Reply:
I also use `pathExists`. `fs-extra` is helpful library because it provides function which are not present in just `fs` and we'll most likely need them anyway. For example `remove` will remove both dir and file. 
Also it provide async versions of `fs` methods out of the box.
My question is how much we are concerned with dependencies? As my understanding we are not building a website so we are not super sensitive to package size.

Reply:
Size it's definitely not an issue in the CLI (it is on lib, since we'll want to add browser support soon), but I'd still be careful in how much we add. If you think it adds value, then let's go for it - I trust your judgement there!